### ISSUE TITLE
IUC packages missing and tool dependency fails

### ISSUE THREAD

**Issue by vick87s** [galaxyproject/galaxy#8594]
I am trying to install mothur however when installing the prior requirements such as legacy and bzlib from IUC - there are errors saying that legacy path doesnt exist and for bzlib make no file found.

How to install the dependencies?

**Comment by martenson** [comment]
@vick87s please provide some log where it went wrong and at least basic configuration of what are you trying to do and where

**Comment by vick87s** [comment]
@martenson - I was trying to install mothur in galaxy and the dependency was bzlib and blast_legacy_2.2.22 (which is an older version of blast) the FTP url being called for during installation is no longer valid as they have moved the installation package from the url mentioned in the installer package of iuc in tool shed.

No logs were created but in manage tools it shows an error stating that the file with the FTP location doesn’t exist

**Comment by luke-c-sargent** [comment]
re: bzip2, looks like they moved their file hosting from the bzip.org url to sourceforge.

the now-dead url was (found [here](https://toolshed.g2.bx.psu.edu/repository/browse_repository?id=8879f09b1dca4ca1#)):
http://www.bzip.org/1.0.6/bzip2-1.0.6.tar.gz

but can now be gotten from:
https://pilotfiber.dl.sourceforge.net/project/bzip2/bzip2-1.0.6.tar.gz

looks like the bzip package needs an update to continue working as-is, but we probably should just have this dependency resolved via conda, right? does this take the form of changing IUC's bzlib package to be conda-based, or should the tools that depend on bzip2 specify that the source is conda?

**Comment by martenson** [comment]
@vick87s unless you are specifically installing some old version of mothur, you shouldn't need to install any of that, everything will be resolved through conda

**Comment by martenson** [comment]
If you want all of the mothur tools, install `suite_mothur` repository. You don't need to install anything prefixed `package_` for the tools to work.

**Comment by vick87s** [comment]
Hi Martin

When I try installing the suite _mothur I get an error stating the page
timed out

**Comment by martenson** [comment]
@vick87s Yeah, that is sadly common with this enormous tool suite when the connection between the given Galaxy and Toolshed is not the best. There are some ways to troubleshoot this described here: https://github.com/galaxyproject/tools-iuc/blob/master/tools/mothur/README#L8

You can also try installing only the selected subtools, and not the whole suite.
